app/web/features/communities/events/EventPage.tsx

```diff
@@ -6,6 +6,7 @@ import {
   Link as MuiLink,
   Theme,
   Typography,
+  Snackbar,
```

Comment:
It looks like we have a Snackbar component already in the `components` folder that has some default pre-configured. Might be worth using that for more consistency?

Reply:
done ✅